parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 NODE_INDEX=$(hostname | tail -c 2)
 NODE_NAME=$(hostname)
-PRIVATE_IP=$(hostname -I | cut -d' ' -f1)
+PRIVATE_IP=$(hostname -i | cut -d' ' -f1)
```

Comment:
It's unclear to me why we're using `-I` (get me all interfaces) vs `-i` get me my primary interface...

Reply:
Note that the ip address here (-i) is a resolved IP address based on the host name.  Is that what you want?

The hostname -I returns all ip addresses without DNS resolution.  This means you get things like loopback, etc.

Reply:
Note that I would look into another way to get the right IP address for the master (like, it must be known somewhere already)